### PR TITLE
Update training and fine-tuning docs

### PR DESCRIPTION
**Pull request by pcuenca**

### CODE REVIEW

File: docs/source/training/dreambooth.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+pip install -U -r diffusers/examples/dreambooth/requirements.txt
+```
+
+Then initialize and configure a [🤗 Accelerate](https://github.com/huggingface/accelerate/) environment with:
+
+```bash
+accelerate config
+```
+
+You need to accept the model license before downloading or using the weights. In this example we'll use model version `v1-4`, so you'll need to visit [its card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree.
```
Comment (patil-suraj) [inline docs/source/training/dreambooth.mdx:47]: Very nice!

File: docs/source/training/overview.mdx [hunk 2]
```diff
@@ -36,13 +36,15 @@ Training examples show how to pretrain or fine-tune diffusion models for a varie
...
+- [Dreambooth](./dreambooth)
 
 
 | Task | 🤗 Accelerate | 🤗 Datasets | Colab
 |---|---|:---:|:---:|
 | [**Unconditional Image Generation**](./unconditional_training) | ✅ | ✅ | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/training_example.ipynb)
-| [**Text-to-Image**](./text2image) | - | - | 
-| [**Text-Inversion**](./text_inversion) | ✅ | ✅ | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/sd_textual_inversion_training.ipynb)
+| [**Text-to-Image fine-tuning**](./text2image) | ✅ | ✅ | 
+| [**Textual Inversion**](./text_inversion) | ✅ | - | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/huggingface/notebooks/blob/main/diffusers/sd_textual_inversion_training.ipynb)
```
Comment (patil-suraj) [inline docs/source/training/overview.mdx:46]: We are not using `datasets` for textual inversion training.

File: docs/source/training/text2image.mdx [hunk 3]
```diff
@@ -11,6 +11,109 @@ specific language governing permissions and limitations under the License.
...
+  --train_batch_size=1 \
+  --gradient_accumulation_steps=4 \
+  --gradient_checkpointing \
+  --mixed_precision="fp16" \
+  --max_train_steps=15000 \
+  --learning_rate=1e-05 \
+  --max_grad_norm=1 \
+  --lr_scheduler="constant" --lr_warmup_steps=0 \
+  --output_dir=${OUTPUT_DIR}
+```
```
Comment (patil-suraj) [inline docs/source/training/text2image.mdx:106]: Let's also add a section for flax example that is just merged #999

**Review by patil-suraj — APPROVED** [review]
Thanks a lot for working on this, looks good to me!

File: docs/source/training/overview.mdx (hunk 2 above)
Comment (pcuenca) [reply docs/source/training/overview.mdx:46]: Yes, I noticed and removed the green tick. The remaining one is for Accelerate.

File: docs/source/training/text2image.mdx (hunk 3 above)
Comment (pcuenca) [reply docs/source/training/text2image.mdx:106]: Good point!

**Review by anton-l — APPROVED** [review]
Very comprehensive guides, thank you Pedro! :heart_eyes:

File: README.md [hunk 4]
```diff
@@ -331,8 +331,25 @@ You can generate your own latents to reproduce results, or tweak your prompt on
...
-  
-## Examples
+
+## Fine-Tuning Stable Diffusion
+
+Fine-tuning techniques make it possible to adapt Stable Diffusion to your own dataset, or add new subjects to it. These are some of the techniques supported in `diffusers`:
+
+Textual Inversion is a technique for capturing novel concepts from a small number of example images in a way that can later be used to control text-to-image pipelines. It does so by learning new 'words' in the embedding space of the pipeline's text encoder. These special words can then be used within text prompts to achieve very fine-grained control of the resulting images. 
+
+- Textual Inversion. Capture novel concepts from a small set of sample images, and associate them to new "words" in the embedding space of the text encoder. Please, refer to [our training examples](https://github.com/huggingface/diffusers/tree/main/examples/textual_inversion) or [documentation](https://huggingface.co/docs/diffusers/training/text_inversion) to try for yourself.
```
[the remark this answers is not in the document]
Comment (pcuenca) [reply README.md]: Thanks!

**Pull request merged**